includes/collection/class-actors.php

```diff
@@ -389,4 +389,255 @@ public static function get_type_by_id( $user_id ) {
+			)
+		);
+
+		if ( ! $post_id ) {
+			return null;
+		}
+
+		return \get_post( $post_id );
+	}
+
+	/**
+	 * Get a remote Actor object by actor URL (guid).
+	 *
+	 * @param int $actor_id The actor ID.
+	 *
+	 * @return \WP_Post|\WP_Error
+	 */
+	public static function get_remote_by_id( $actor_id ) {
+		return \get_post( $actor_id );
+	}
+
+	/**
+	 * This function is used to store errors that occur when
+	 * sending an ActivityPub message to a Follower.
+	 *
+	 * The error will be stored in post meta.
+	 *
+	 * @param int   $post_id The ID of the WordPress Custom-Post-Type.
+	 * @param mixed $error   The error message. Can be a string or a WP_Error.
+	 *
+	 * @return int|false The meta ID on success, false on failure.
+	 */
+	public static function track_error( $post_id, $error ) {
+		if ( \is_string( $error ) ) {
+			$error_message = $error;
+		} elseif ( \is_wp_error( $error ) ) {
+			$error_message = $error->get_error_message();
+		} else {
+			$error_message = \__(
+				'Unknown Error or misconfigured Error-Message',
+				'activitypub'
+			);
+		}
+
+		return \add_post_meta(
+			$post_id,
+			'_activitypub_errors',
+			$error_message
+		);
+	}
+
+	/**
+	 * Count the errors for a Follower.
+	 *
+	 * @param int $post_id The ID of the WordPress Custom-Post-Type.
+	 *
+	 * @return int The number of errors.
+	 */
+	public static function count_errors( $post_id ) {
+		return \count( \get_post_meta( $post_id, '_activitypub_errors', false ) );
+	}
+
+	/**
+	 * Clear the errors for a Follower.
+	 *
+	 * @param int $post_id The ID of the WordPress Custom-Post-Type.
+	 *
+	 * @return bool True on success, false on failure.
+	 */
+	public static function clear_errors( $post_id ) {
+		return \delete_post_meta( $post_id, '_activitypub_errors' );
+	}
+
+	/**
+	 * Get all Followers that had errors.
+	 *
+	 * @param int $number Optional. The number of Followers to return. Default 20.
+	 *
+	 * @return \WP_Post[] The Term list of Followers.
+	 */
+	public static function get_faulty( $number = 20 ) {
+		$args = array(
+			'post_type'      => self::POST_TYPE,
+			'posts_per_page' => $number,
+			// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+			'meta_query'     => array(
+				'relation' => 'OR',
+				array(
+					'key'     => '_activitypub_errors',
+					'compare' => 'EXISTS',
+				),
+				array(
+					'key'     => '_activitypub_inbox',
+					'compare' => 'NOT EXISTS',
+				),
+				array(
+					'key'     => '_activitypub_inbox',
+					'value'   => '',
+					'compare' => '=',
+				),
+			),
+		);
+
+		$posts = new \WP_Query( $args );
+		return $posts->get_posts();
+	}
+
+	/**
+	 * Get all Followers that have not been updated for a given time.
+	 *
+	 * @param int $number     Optional. Limits the result. Default 50.
+	 * @param int $older_than Optional. The time in seconds. Default 86400 (1 day).
+	 *
+	 * @return \WP_Post[] The Term list of Actors.
+	 */
+	public static function get_outdated( $number = 50, $older_than = 86400 ) {
```

Comment:
```suggestion
	 * @param int $older_than Optional. The time in seconds. Default DAY_IN_SECONDS.
	 *
	 * @return \WP_Post[] The list of Actors.
	 */
	public static function get_outdated( $number = 50, $older_than = DAY_IN_SECONDS ) {
```
Term might be confusing? It confused me since I could see anything related to terms in the function :)